### ISSUE TITLE
Count duplicates and delete them when icon is clicked

### ISSUE THREAD

**Issue by allanwind** [igorlogius/dont-open-duplicate-tabs#13]
**Is your feature request related to a problem? Please describe.**

I like the minimal nature of the add-on but the "show remove notification" isn't optimal either set or unset.  If set you get interrupted with a pop-up that you have to dismiss.  If unset you have pay close attention notice that one of your tabs was just closed. Both demand your attention now but the only cost of delaying action is extra memory usage.

**Describe the solution you'd like**

Would it be possible for the add-on to simply count number of duplicates, show if >0 (instead of on), and when you click the icon it would delete the duplicate the duplicates like now?

**Describe alternatives you've considered**

Duplicate Tab Close works similar but I suspect is has a bug that cause all UI dialogs to be auto-closed right away.  In either case, it is kinda bloated, and you have to click on the icon, wait for it show the tabs, and only then can you close it.  FoxyTab, TabSearch all lack the visual indication of duplicate tabs.

**Additional context**

**Comment by igorlogius** [comment]
> Would it be possible for the add-on to simply count number of duplicates, show if >0

Yes, that would be possible ... but its not so easy to add.
I'll have to see what i can do.

**Comment by igorlogius** [comment]
@allanwind  

Take a look at this: https://addons.mozilla.org/en-US/firefox/addon/duplicate-tabs-closer/

**Comment by allanwind** [comment]
Yeah, that is what I have been using, but I think it's triggering 
a defect causing all dialogs to auto-close.  This means I have to
restart Firefox frequently, say, after 10 minutes if I need a 
dialog.  It's also rather slow as you click the icon, wait for it 
to populate the list of tabs (could be seconds when I have a few 
hundred open), then scroll down (font/ux issue) to to finally 
click close duplicates.


/Allan

On 2023-03-16T11:15:51, igorlogius wrote:
>@allanwind
>
>Take a look at this: https://addons.mozilla.org/en-US/firefox/addon/duplicate-tabs-closer/
>
>
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/igorlogius/dont-open-duplicate-tabs/issues/13#issuecomment-1472491413
>You are receiving this because you were mentioned.
>
>Message ID: ***@***.***>

-- 
Allan Wind
Yaxto - Runs Your Business
<https://yaxto.com>

**Comment by igorlogius** [comment]
@allanwind 

Do i undestand you correct in that you dont need the automatic removal? 

You are just interested in knowing the amount of duplicates and manually triggering the removal. - Is that correct?
